### ISSUE TITLE
Vamp doesn't figure out that sava 1.0 is already running 

### ISSUE THREAD

**Issue by jgordijn** [magneticio/vamp#228]
The documentation states:
>It is our goal to update the already running deployment with the new blueprint. Vamp will figure out that v1.0.0 is already there and just add v1.1.0 while setting the correct routing between these services.

However when I follow the tutorial I end up with the following situation:
![screenshot 2015-06-26 08 51 11](https://cloud.githubusercontent.com/assets/210142/8372150/cd4aa942-1be0-11e5-9dff-e5be0f320c77.png)

As you can see, the 1.0 version is deployed 2 times, while the `instances` variable in the blueprint is 1.

What happens to the routing? It should distribute evenly among 1.0 and 1.1.

**Comment by jgordijn** [comment]
And after step 3 I end up with:

![screenshot 2015-06-26 09 03 29](https://cloud.githubusercontent.com/assets/210142/8372303/5587fb6a-1be2-11e5-8471-27a72d9ed33e.png)

**Comment by tnolet** [comment]
Hi @jgordijn 

I replayed the tutorial and was not able to reproduce this situation. So there are two options:
1. Either this is a bug in the currently published version of our Docker distribution that we solved already in the one that I am using. 
2. You maybe stop or restarted Vamp in between these actions. 

In either case, I get the correct results with out latest build and we have fixed some critical issues in the Docker driver since the former, 0.7.7 release.
